apps/torch/app.js

```diff
@@ -1,21 +1,34 @@
-const SETTINGS_FILE = "torch.json";
-let settings;
+{
+  const SETTINGS_FILE = "torch.json";
+  let settings;
 
-function loadSettings() {
-  settings = require("Storage").readJSON(SETTINGS_FILE,1)|| {'bg': '#FFFFFF', 'color': 'White'};
-}
+  let loadSettings = function() {
+    settings = require("Storage").readJSON(SETTINGS_FILE,1)|| {'bg': '#FFFFFF', 'color': 'White'};
+  };
 
-loadSettings();
+  loadSettings();
+  let wid_utils = require("widget_utils");
 
-Bangle.setLCDBrightness(1);
-Bangle.setLCDPower(1);
-Bangle.setLCDTimeout(0);
-g.reset();
-g.setTheme({bg:settings.bg,fg:"#000"});
-g.setColor(settings.bg);
-g.fillRect(0,0,g.getWidth(),g.getHeight());
-Bangle.setUI({
-  mode : 'custom',
-  back : load, // B2: SW back button to exit
-  btn : ()=>{load();}, // B1&2: HW button to exit. // A simple 'load' as on the line above did not work for btn???
-});
+  let brightnessBackup = require("Storage").readJSON('setting.json').brightness;
```

Comment:
A general note: Should brightness be part of the options returned by Bangle.getOptions() instead?